.github/workflows/pr.yml

```diff
@@ -0,0 +1,19 @@
+on: pull_request
+name: Review
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    name: Changelog should be updated
+    strategy:
+      fail-fast: false
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Git fetch
+        run: git fetch
+
+      - name: Check that changelog has been updated.
+        run: git diff --exit-code origin/develop -- CHANGELOG.md && exit 1 || exit 0
```

Comment:
We should use `${{ github.base_ref }}` (cf. https://docs.github.com/en/actions/learn-github-actions/contexts#github-context) rather than `develop` here since we may be making a pull request against a branch other than `develop` (see https://github.com/itk-dev/os2forms_selvbetjening/blob/develop/.github/workflows/pr.yaml#L19 for an example use).